run_scraper_data.sh

```diff
@@ -0,0 +1,2 @@
+source env/bin/activate;
+python3 scraper_data.py; # how do we pass command line args to python? 
```

Comment:
FYI, you can pass all arguments along via:

```suggestion
python3 scraper_data.py $@;
```

(In bash, `$@` is a list of all arguments, `$#` is the number of arguments, and `$1`, `$2`, etc. are the individual arguments.)